docs/backup_and_restore.rst

```diff
@@ -101,89 +88,407 @@ When you are ready to begin the backup, run
+      output, we recommend that you contact us immediately at
+      securedrop@freedom.press 
+      (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
+
+#. Copy the old instance's configuration files and backup from 
+   ``~/Persistent/sd.bak`` into ``~/Persistent/securedrop`` using the following 
+   Terminal commands:
+
+   .. code:: sh
+
+      cd ~/Persistent/securedrop
+      export SD_OLD="~/Persistent/sd.bak/install_files/ansible-base"
+      export SD_NEW="~/Persistent/securedrop/install_files/ansible-base"
+      cp $SD_OLD/group_vars/all/site-specific $SD_NEW/group_vars/all/site-specific
+      cp $SD_OLD/tor_v3_keys.json $SD_NEW/tor_v3_keys.json
+      cp $SD_OLD/sd-backup-old.tar.gz $SD_NEW/sd-backup-old.tar.gz
+
+   You will also need to copy the old instance's *Submission Public Key*,
+   *OSSEC Alert Public Key*, and, if configured, the *Journalist Alert Public Key*.
+   Assuming that these are named ``SecureDrop.asc``, ``ossec.asc``, and
+   ``journalist.asc`` respectively, run the following commands:
+
+   .. code:: sh
+
+      cp $SD_OLD/SecureDrop.asc $SD_NEW/SecureDrop.asc
+      cp $SD_OLD/ossec.asc $SD_NEW/ossec.asc
+      cp $SD_OLD/journalist.asc $SD_NEW/journalist.asc
+
+#. *(Optional):* If your old instance was configured to provide the *Source
+   Interface* via HTTPS, you should also copy across the certificate, certificate
+   key, and chain file. Assuming that these are named ``sd.crt``, ``sd.key``, and
+   ``ca.crt`` respectively, run the following commands:
 
-Make sure to replace ``sd-backup-2017-07-22--01-06-25.tar.gz`` with the filename
-for your backup archive. The backup archives are located in
-``install_files/ansible-base``.
+   .. code:: sh
 
-Once the restore is done, the *Application Server* will use the original
-*Source Interface* and *Journalist Interface* Onion URLs. You will need
-to update the corresponding files on the *Admin Workstation*:
+      cp $SD_OLD/sd.{crt,key} $SD_NEW/
+      cp $SD_OLD/ca.crt $SD_NEW/
 
-* ``app-source-ths``
-* ``app-journalist-aths``
-* ``app-ssh-aths``
+#. Install Ubuntu 20.04 on the *Application* and *Monitor Servers*, following
+   the :doc:`server setup instructions<servers>` to install with the correct
+   settings, test connectivity, and set up SSH keys to allow for
+   *Admin Workstation* access.
 
-Once SSH access to the servers has been established (or if using SSH over
-local network), Onion URLs for the *Source Interface* and *Journalist Interfaces*
-can be fetched using the installer:
+#. Reinstall SecureDrop on the servers, following the :doc:`installation
+   instructions <install>`. During the configuration stage
+   (``./securedrop-admin sdconfig``), the values will be prepopulated based on
+   the old instance's configuration. Press **Enter** to accept each value,
+   except when you are asked if you want to enable v2 onion services - instead,
+   type ``no``.
```

Comment:
This section includes v3-only backups, in which case they won't have to type ``no``. I think that's reasonably obvious in context, but leaving a note in case you want to futz with the wording.

Reply:
addressed in  7f2b421